lang/java/avro/src/main/java/org/apache/avro/LogicalTypes.java

```diff
@@ -81,8 +86,11 @@ private static LogicalType fromSchemaImpl(Schema schema, boolean throwErrors) {
         logicalType.validate(schema);
       }
     } catch (RuntimeException e) {
+      LOGGER.debug("Invalid logical type found", e);
       if (throwErrors) {
         throw e;
+      } else {
+        LOGGER.warn("Ignoring invalid logical type for name: {}", typeName);
```

Comment:
Ah sorry, one more minor issue. All the lines below this execute if throwErrors is false, but only the `LOGGER.warn` line is the else branch, which is a bit confusing/inconsistent. Could you move it out of the else, immediately after the if block like the existing code?